app/scripts/convert.py

```diff
@@ -0,0 +1,71 @@
+# convert an spreadsheet-exported json into one that can be used in a django migration
+# get path and read json file
+# get model path
+# write new json with model and fields
+
+# example:
+# docker-compose exec web python core/scripts/convert.py core/fixtures/userstatus_export.json
+# to apply the seed script:
+# docker-compose exec web python manage.py runscript userstatus-seed
+
+import json
+import sys
+from pathlib import Path
+
+
+def get_modelname(path):
+    filename = Path(path).name
+    return filename.split("_")[0]
+
+
+def to_key_value(_input):
+    # print(_input)
```

Comment:
Please remove all the extraneous comments and print statements

Reply:
Implementing